texar/modules/networks/conv_networks.py

```diff
@@ -531,3 +533,19 @@ def _infer_dense_layer_input_size(self, input: torch.Tensor) -> torch.Size:
         with torch.no_grad():
             output = super().forward(input)
         return output.view(output.size()[0], -1).size()
+
+    @property
+    def output_size(self) -> int:
+        r"""Return the finale dimension of :meth:`forward` output tensor.
+        """
+        # TODO: We will change the implementation to
+        # something that does not require a forward pass.
+        if self._in_features is not None:
+            input = torch.Tensor(1, self._in_channels, self._in_features)
+        else:
+            kernel_size = self._hparams.kernel_size
```

Comment:
Do we have to do a forward pass here? `input_size` to the dense layers are computed in `__init__` via a forward pass, can use just reuse that result?